playbooks/roles/common-deployer/templates/helm-serve.service.j2

```diff
@@ -5,7 +5,7 @@ After=network.target
 [Service]
 User={{ ansible_user }}
 Restart=always
-ExecStart=/usr/local/bin/helm serve
+ExecStart=/usr/bin/helm serve
```

Comment:
Im wondering if we should just put the helm-serve service directly on the package so it can be used as is by other package users?

Reply:
Oh, that was the plan. See https://build.opensuse.org/request/show/703661
Seems like I forgot to remove the template after all